engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -759,7 +759,10 @@ object Main {
         case Some(main) if mainMethodName != "main" =>
           main.execute(mainType.newInstance())
         case Some(main) =>
-          main.execute()
+          val res = main.execute()
+          if (!res.isNull()) {
+            err.println(res);
```

Comment:
Why do we print to stderr instead of stdout?

Reply:
OK, will change the code to print to `out`.